core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -506,7 +506,8 @@ UNWIND(	.cantunwind)
 
 #if defined (CFG_BOOT_SECONDARY_REQUEST)
 	/* if L1 is not invalidated before, do it here */
-	bl	arm_cl1_d_invbysetway
+	mov	r0, #DCACHE_OP_INV
+	bl	dcache_op_all
```

Comment:
Good question, I don't know. On the other hand this is done once during boot so it's not that much time that can be saved by doing it only to louis.